lib/auth/auth_with_roles.go

```diff
@@ -2258,6 +2258,30 @@ func (a *ServerWithRoles) DeleteAuthPreference(context.Context) error {
 	return trace.NotImplemented(notImplementedMessage)
 }
 
+// GetClusterAuditConfig gets cluster audit configuration.
+func (a *ServerWithRoles) GetClusterAuditConfig(ctx context.Context, opts ...services.MarshalOption) (types.ClusterAuditConfig, error) {
+	if err := a.action(defaults.Namespace, types.KindClusterAuditConfig, types.VerbRead); err != nil {
```

Comment:
Once you merge/rebase master, `lib/defaults.Namespace` won't be defined. You'll need to replace it with apidefaults here and elsewhere.
```suggestion
	if err := a.action(apidefaults.Namespace, types.KindClusterAuditConfig, types.VerbRead); err != nil {
```